### DISCUSSION TITLE
Tauri Action: Updater latest.json file

### DISCUSSION THREAD

**Discussion by TristanPouliquen** [tauri-apps/tauri#6385] [Q&A] [answered]
Hello,

I tried to enable the Updater feature the other day. I configured my `tauri.conf.json` (https://github.com/TristanPouliquen/selection-poste-enm/blob/main/src-tauri/tauri.conf.json#L58), and added my Tauri signing keys to my Github workflow.

The workflow is working rather correctly, as can be seen on the latest release: https://github.com/TristanPouliquen/selection-poste-enm/releases/tag/v0.2.0

I have all bundled files for MacOS, Windows & Linux and the `.tar.gz`, `.zip`, `.sig` assets that are linked to the updater feature.

My problem is I don't see the `latest.json` file containing the json static content to make my apps load the new version automatically.

I read the action source code, and particularly https://github.com/tauri-apps/tauri-action/blob/dev/src/upload-version-json.ts, but I do not understand what is going wrong.

In my action output, I do not get any error output from updating the `latest.json` file that I can see in the action source code.

Here is my last workflow run: https://github.com/TristanPouliquen/selection-poste-enm/actions/runs/4306817724

Is there something I missed in the setup of my workflow?

https://github.com/TristanPouliquen/selection-poste-enm/blob/main/.github/workflows/build_release.yml

Thank you in advance!

**Comment by FabianLars** [comment, marked as answer]
It's not released yet and only works if you use the action from git like `tauri-apps/tauri-action@dev` (you can also use commit hashes to lock it to a commit)

**Reply by TristanPouliquen** [reply]
Right, I'll try to use it at @dev to see how it works then, thank you :) 

I saw the PR adding it but I didn't look at the date to see if it was recent or not

**Comment by bartosz-skejcik** [comment]
Is it released? I'm new to tauri and i want to add the updater functionality to [my app](https://github.com/bartosz-skejcik/content-planner).

**Reply by FabianLars** [reply]
Yes. You just have to configure your app to use the updater and the action should generate the latest.json file for you. tauri-action usually does not require any additional configuration (edit: i mean in the workflow file).

**Reply by bartosz-skejcik** [reply]
oohh. thats perfect. I have one more question tho. How do i add it to my repo? cause currently in my `tauri.conf.json` i have something like this:
```json
    "updater": {
      "pubkey": "dW50c.....",
      "windows": {
        "installMode": "basicUi"
      },
      "endpoints": [
        "https://raw.githubusercontent.com/bartosz-skejcik/content-planner/refs/heads/release/latest.json"
      ]
    }
```

but obviously, the file which is added to the `endpoints` doesnt exist yet. after the first successful release with the updater i hope it will be there hahah

(edit: or where should i point the edpoint to if the tauri-action generates the `latest.json` file?)

**Reply by FabianLars** [reply]
> (edit: or where should i point the edpoint to if the tauri-action generates the latest.json file?)

Typically `https://github.com/bartosz-skejcik/content-planner/releases/latest/download/latest.json`
